src/pages/index.module.scss

```
@@ -65,7 +65,7 @@
   display: block;
   background-color: var(--card-color);
   border: rem(1px) solid var(--card-border-color);
-  border-radius: rem(8px);
+  border-radius: rem(5x);
```

Stylelint (17.14.1)

[error] Unknown unit "x" (unit-no-unknown)

(unit-no-unknown)